temoa/model_checking/network_model_data.py

```
@@ -1,348 +1,370 @@
-        """create a copy of the current"""
-        return NetworkModelData(
-            demand_commodities=self.demand_commodities.copy(),
-            waste_commodities=self.waste_commodities.copy(),
-            source_commodities=self.source_commodities.copy(),
-            capacity_commodities=self.capacity_commodities.copy(),
-            exchange_commodities=self.exchange_commodities.copy(),
-            all_commodities=self.physical_commodities.copy(),
-            available_techs=self.available_techs.copy(),
-            available_linked_techs=self.available_linked_techs.copy(),
-        )
+TechAttributeValue: TypeAlias = ParameterValue
```

Ruff (0.14.1)

Type alias `TechAttributeValue` uses `TypeAlias` annotation instead of the `type` keyword

Use the `type` keyword

(UP040)